### DISCUSSION TITLE
Is it possible to use the serverless API with gridsome?

### DISCUSSION THREAD

**Discussion by pmochine** [vercel/vercel#5150] [Help] [answered]
Sorry for this noob question, but I'm new with JamStack.

So I've written a website with the framework gridsome. Now I am curious if it is still possible to add an API folder and write the serverless function in node.js. 

If yes, how do you deal with the package.json? Since now it's going to be a mix between node.js and gridsome? Or is it possible to have two seperate package.json files? One in the root and the other in `/api`?

**Comment by styfle** [comment, marked as answer]
Hi @pmochine 

Yes you can add [Serverless Functions](https://vercel.com/docs/serverless-functions/introduction) in the `/api` directory and use Gridsome for the frontend SSG.

Yes, you can use a single `package.json` in the root or have a separate `/api/package.json` for your Serverless Functions.
